main.py

```diff
@@ -26,14 +26,29 @@
 
 
 def main(is_simulation: bool, real_servo: bool) -> None:
+    """
+    The main function that will be run when the program is started. It will create the objects that will be used in the
+    airbrakes context and run the main loop. The main loop will run until the user presses Ctrl+C.
+
+    This function is what handles if the program is running in simulation mode or not.
```

Comment:
sorry, I still don't understand what you're trying to say. main() does not "handle" whether the program runs in sun mode or not. That's defined by the parameter real_time_sim